grpc/src/main/java/com/linecorp/armeria/client/grpc/GrpcClientOptions.java

```diff
@@ -78,8 +78,9 @@
      * work. If {@link GrpcUnsafeBufferUtil#releaseBuffer(Object, RequestContext)} is not called, the memory
      * will be leaked.
      *
-     * <p>Note that this isn't working if the payloads are compressed or the {@link SerializationFormat} is
-     * {@link GrpcSerializationFormats#PROTO_WEB_TEXT}.
+     * <p>Note that this has no effect if the payloads are compressed or the {@link SerializationFormat} is
+     * {@link GrpcSerializationFormats#PROTO_WEB_TEXT}, so you should not call
```

Comment:
Is this `should not` or `do not need to`? It'd be nice if nothing bad happens the `releaseBuffer()` is called.

Reply:
It raises an exception if it's called.
Let me just return boolean instead of raising an exception. 😉